### ISSUE TITLE
"Are you sure?" when closing "busy" browser UI window

### ISSUE THREAD

**Issue by FumbleFinger5** [sc0ty/subsync#127]
Many thanks for a great piece of software!

I know it looks a bit trivial, but imho this would be a seriously "user-friendly" tweak!

Some web pages (usually, *porn* sites! :) display an "Are you sure?" pop-up when you try to close the page. That's never been anything more than an annoyance to me, but after having accidentally closed the subsync Web UI page several times (thereby losing a partly or fully re-synced subtitle not yet downloaded to my PC) I can really see why it would be good if web-based subsync could prompt for confirmation before closing its browser window (IF it's either still processing, OR has finished, but not yet copied the result back to the requesting PC).

**Comment by sc0ty** [comment]
Good idea. I'll have to clean up web build scripts first, unfortunately.

**Comment by FumbleFinger5** [comment]
I can wait! Thanks for the acknowledgement, anyway.

Also note that I just "tipped" you 10 BAT cryptotokens from my Brave browser, but you don't seem to be set up to receive them. It might be worth you registering with Brave to receive their crypto-tips, as I'm probably not the only person trying to give them to you. I think Brave hold unclaimed tips for 3-6 months before "burning" them out of existence.
